### ISSUE TITLE
MarkInfo objects are deprecated since Pytest-3.6.0

### ISSUE THREAD

**Issue by daa** [ionelmc/pytest-benchmark#122]
``pytest-3.6.0`` issues following warning:
```
site-packages/pytest_benchmark/plugin.py:392: RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.
Please use node.get_closest_marker(name) or node.iter_markers(name).
Docs: https://docs.pytest.org/en/latest/mark.html#updating-code
  options = marker.kwargs if marker else {}
```
- Release notes: https://docs.pytest.org/en/latest/changelog.html#pytest-3-6-0-2018-05-23
- Code update information: https://docs.pytest.org/en/latest/mark.html#marker-revamp-and-iteration

**Comment by fizyk** [comment]
and with pytest 4 it now throws an error.

**Comment by ionelmc** [comment]
Should be fixed by a464d44144be264105a9e1930b69ee209ea261f4.
